lib/pod/command/keys/set.rb

```diff
@@ -40,16 +39,16 @@ def run
           # info "Saving into the keychain."
 
           keyring = current_keyring
-          keyring.keys << @key_name.gsub("-", "_")
+          keyring.keys << @key_name.tr('-', '_')
```

Comment:
It's simpler (and faster) when you're just changing one character -- `gsub` on the other hand has the full power of regular expressions